### DISCUSSION TITLE
TextDecoder error when install cardano-wallet-js

### DISCUSSION THREAD

**Discussion by Sotatek-HienPham3** [mastodon/mastodon#25384] [Q&A]
I get an error when installing  **cardano-wallet-js** library into mastodon source.
When I import import { WalletClient } from 'cardano-wallet-js'; to the component, the page is crashed
![image](https://github.com/mastodon/mastodon/assets/93501321/a3f0bec1-5ed6-43de-877d-db8706190cbd)

**Comment by nightpool** [comment]
Mastodon cannot provide support for third party extensions, they may break the page or cause it to crash in undefined ways.
